opendevin/memory/condenser.py

```diff
@@ -1,26 +1,196 @@
 from opendevin.core.logger import opendevin_logger as logger
+from opendevin.events.action.agent import (
+    AgentFinishAction,
+    AgentRecallAction,
+    AgentSummarizeAction,
+)
+from opendevin.events.action.browse import BrowseInteractiveAction, BrowseURLAction
+from opendevin.events.action.commands import (
+    CmdKillAction,
+    CmdRunAction,
+    IPythonRunCellAction,
+)
+from opendevin.events.action.empty import NullAction
+from opendevin.events.action.files import FileReadAction, FileWriteAction
+from opendevin.events.action.message import MessageAction
+from opendevin.events.event import Event, EventSource
+from opendevin.events.observation.observation import Observation
+from opendevin.events.serialization.event import event_to_memory
 from opendevin.llm.llm import LLM
+from opendevin.memory.history import ShortTermHistory
+from opendevin.memory.prompts import get_summarize_prompt, parse_summary_response
+
+MAX_USER_MESSAGE_CHAR_COUNT = 200  # max char count for user messages
```

Comment:
User messages can definitely be longer than 200 characters. Is there some justification for this limit?

Reply:
It's used to summarize a user message that is longer than 200 characters, and not a likely task. That is, it's to avoid summarizing those eligible _and smaller_ than that.

It's because I noticed that summarizing small messages is practically pointless... the summary might be larger than the message! 😅

Reply:
Ah, sorry, understood!